### PR TITLE
WIP: Add data as json

### PR DESCRIPTION
**Pull request by RAlanWright**
- [x] Grab csv from Google Docs
- [x] Convert csv to json
- [x] Doublecheck json is correctly structured
- [ ] Change fields to 1 word
- [ ] ?

### CODE REVIEW

**Review by xanderyzwich — APPROVED** [review]
This looks like it does the job

**Comment by xanderyzwich** [comment]
closes #1

**Comment by RAlanWright** [comment]
@xanderyzwich @Capocaccia  Should we create a separate issue solely for getting this static data in, rather than closing #1 which includes 2 other things?

**Review by Kaytal — APPROVED** [review]
This is a good straight data to JSON migration.  In the future (and maybe I'll open an issue for it) we should probably structure this differently... Just food for thought.

**Comment by Capocaccia** [comment]
@RAlanWright How comfortable are you getting this data via API? I think we are now leaning towards this being a better method.

https://hooshmand.net/use-google-sheets-json-database/

File: src/App.tsx [hunk 1]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 
+// Testing data output
+const SalaryData = require('./data/MemtechSalaryData.json');
+
+console.log(SalaryData);
```
Comment (Capocaccia) [inline src/App.tsx]: Remove console log please.

**Review by Capocaccia — APPROVED** [review]

**Pull request merged**